agent/tool-scripts/prometheus-metrics

```diff
@@ -174,15 +171,9 @@ case "$mode" in
 		# In case the file doesn't have certs, port and they are not defined while registering the tool, the default values are set.
 		# host port=<port> cert=<cert> key=<key>
 		host=$(echo $file | awk -F " " '{ print $1 }')
-		if [[ -z "$port" ]]; then
-			port=$(echo $file | grep -w "port" | awk -F " " '{ print $2 }' | awk -F "=" '{print $2}')
-		fi
-		if [[ -z "$cert" ]]; then
-			cert=$(echo $file | grep -w "cert" | awk -F " " '{ print $3 }' | awk -F "=" '{print $2}')
-		fi
-		if [[ -z "$key" ]]; then
-			key=$(echo $file | grep -w "key" | awk -F " " '{ print $4 }' | awk -F "=" '{print $2}')
-		fi
+		port=$(echo $file | grep -w "port" | awk -F " " '{ print $2 }' | awk -F "=" '{print $2}')
```

Comment:
Should the absence of a port spec in the inventory file be flagged as an error instead of using a default port? Does your doc explain all the permutations?